COPYING

```diff
@@ -1,6 +1,19 @@
 This repository contains the generator for the MAVLink protocol. The generator itself is
-(L)GPL v3 licensed, while the generated code is subject to different licenses:
-
+(L)GPL v3 licensed, while the generated code is subject to the MIT license.
+
+Copyright 2010-2019
```

Comment:
There really is no reason to put some names holding copyright here. If copyright names are wanted to be tracked then a CONTRIBUTORS.md file should be created with **everyone** inside.

Reply:
For Ardupilot we have 412 Contributors, many of them with a username, but not a proper name.
Is there a way of automaticaly collect all of them and place them in a generated file?

I simply added the names of the 20 to contributors here, but I did it by hand, tracking down the real names behind the usernames.

Reply:
> For Ardupilot we have 412 Contributors, many of them with a username, but not a proper name.

That's life when you don't track down the contributors, which we don't.

> Is there a way of automaticaly collect all of them and place them in a generated file?

Likely there are tools available for that. However, at least in ArduPilot, that is unlikely to collect all contributors as code was imported into Git from somewhere else.

> I simply added the names of the 20 to contributors here, but I did it by hand, tracking down the real names behind the usernames.

Besides the top 20 changing, it isn't fair to the other contributors. At a minimum, we should allow other contributors to be added to the file - and with the number of them, we clearly need a separate file.

Also, you seem to be concerned with the Volkswagen legal department: I don't see how listing a very small number of copyright holders is acceptable.

Reply:
> Also, you seem to be concerned with the Volkswagen legal department: I don't see how listing a very small number of copyright holders is acceptable.

Well most FOSS projects only list a single copyright owner, so by listing 20 we are already on the safe side.